apps/blog-front/layouts/default.vue

```diff
@@ -22,8 +30,9 @@
         titleTemplate: `%s | ${this.config.titleSuffix}`,
         meta: [
           { hid: 'description', name: 'description', content: this.config.description },
-          { hid: 'keywords', name: 'keywords', content: this.config.keywords }
-        ]
+          { hid: 'keywords', name: 'keywords', content: this.config.keywords },
+        ],
+        link
```

Comment:
I did not understand what happens if link  isn't being filled (this.config.themeStylesUrl === false on line 18)

Reply:
The "link" property can be an empty array.
Each item in this array will represent a "link" tag on the HTML file.